src/app/dataset-view/additional-components/dataset-settings-component/tabs/webhooks/rotate-secret-subscription-modal/rotate-secret-subscription-modal.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="d-flex p-4">
+    <div class="flex-grow-1 ms-2">
+        <span class="fs-5 fw-bold">{{ "Webhook subscription rotate secret" }}</span>
+    </div>
+    <div>
+        <button
+            type="button"
+            class="btn-close"
+            aria-label="Close"
+            (keydown.esc)="activeModal.close()"
+            (click)="activeModal.close()"
+        ></button>
+    </div>
+</div>
+<div class="px-4">
+    <mat-divider />
+</div>
+
+<div class="body p-4">
+    <div class="d-flex flex-column">
+        <label for="" class="form-label">Secret </label>
+        <div class="d-flex flex-row mb-3">
+            <input
+                type="text"
+                data-test-id="subscription-secret-rotate"
+                [(ngModel)]="secretRotate"
```

Comment:
Do we need 2-way binding for a read-only value?

Reply:
we need 1-way binding value, fixed